### PR TITLE
Resolve issues in API documentation 

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* Fixed some grammatical and formatting mistakes
* Updated the page descriptions

CC: @Nishchit14 @vipulgupta2048

### CODE REVIEW

**Comment by Sc0rpi0n101** [comment]
The `Writing GraphQL queries` page is not rendering.

image

**Comment by vipulgupta2048** [comment]
Great catch @Sc0rpi0n101 :+1: 

@Nishchit14 I have a deployed a HOTFIX for this on the same PR. It should resolve it now :crossed_fingers:

File: source/apis/impressions/query-parameters.mdx [hunk 1]
```diff
@@ -10,16 +10,16 @@ description: Learn about the Query Parameters for your APIs
...
 For example, breaking down the URL (Uniform Resource Locator) https://jsonplaceholder.typicode.com/posts?id=12&name=vipulgupta2048
 
-The protocol is https
-The domain name is jsonplaceholder.typicode.com
-The path to the resource (Path Parameter) is /posts
-Finally, the query parameters being id=12&name=vipulgupta2048
+The protocol is *https*  
+The domain name is *jsonplaceholder.typicode.com*  
+The path to the resource (Path Parameter) is */posts*  
+Finally, the query parameters being *id=12&name=vipulgupta2048*
```
Comment (vipulgupta2048) [inline source/apis/impressions/query-parameters.mdx:16]: I think bold would look better. Go bold!

File: source/apis/impressions/query-parameters.mdx [hunk 2]
```diff
@@ -10,16 +10,16 @@ description: Learn about the Query Parameters for your APIs
...
+The domain name is *jsonplaceholder.typicode.com*  
+The path to the resource (Path Parameter) is */posts*  
+Finally, the query parameters being *id=12&name=vipulgupta2048*  
 
 Query Parameters are critical in sorting or filtering through resources for developers to find the right data. One can specify query parameters easily using the `Params` tab in the request panel of the API app.
 
 > PLACEHOLDeR FOR IMAGE of adding params
 
-Adding query parameters at one place in the API app will lead to them being added to the URL bar instantly.
+Adding query parameters at any place in the API app will lead to them being added to the URL bar instantly.
```
Comment (vipulgupta2048) [inline source/apis/impressions/query-parameters.mdx:22]:
```suggestion
Adding query parameters in the API app will result in them being added instantly to the URL present in the URL bar as parameters
```

File: source/apis/impressions/url-bar.mdx [hunk 3]
```diff
@@ -19,15 +20,15 @@ The URL bar consists of 5 elements,
...
+- **POST** - The `POST` method is used to submit an entity to the specified resource, often causing a change in state or modify the data.
 - **PATCH** - The `PATCH` method is used to apply partial modifications to a resource.
 - **PUT** - The `PUT` method replaces all current representations of the target resource with the request payload.
 - **DELETE** - The `DELETE` method deletes the specified resource.
 - **HEAD** - The `HEAD` method asks for a response identical to that of a `GET` request, but without the response body.
 - **OPTION** - The `OPTION` describes the communication options for the target resource.
 - **PURGE** - The `PURGE` is used for removing cached files from the server, but can be used for removing anything related to caching.
 - **LINK** - The `LINK` method adds meta-information (Object header information) to an object, without modifying the object's content.
-- **UNLINK** - This `UNLINK` method deletes metainformation about an object. Only headers exactly matching the headers given are removed.
+- **UNLINK** - This `UNLINK` method deletes meta-information about an object. Only headers exactly matching the headers given are removed.
```
Comment (vipulgupta2048) [inline source/apis/impressions/url-bar.mdx:31]:
```suggestion
- **UNLINK** - The `UNLINK` method deletes meta-information about an object. Only headers exactly matching the headers given are removed.
```

**Comment by Nishchit14** [comment]
@vipulgupta2048 Once you approve the changes then I'll merge it immediately.

**Comment by Nishchit14** [comment]
Awesome!! This looks great to me @Sc0rpi0n101 

I will review it locally again by deploying doc and then merge it.

**Pull request merged**